webpack/webpack.browser.config.js

```diff
@@ -0,0 +1,19 @@
+const path = require('path');
+const webpack = require('webpack');
+
+const TerserPlugin = require('terser-webpack-plugin');
```

Comment:
I didn’t see this added in package.json?

Reply:
Removed.